tools/testing/target_determination/heuristics/interface.py

```diff
@@ -0,0 +1,86 @@
+from abc import abstractmethod
+from typing import Any, Dict, List
+
+
+class TestPrioritizations:
+    """
+    Describes the results of whether heuristics consider a test relevant or not.
+    Heuristics can leave a list empty if they don't consider any tests relevant.
+
+    Heuristics can leave the unranked_relevance list empty to imply all unmentioned tests are irrelevant
+    """
+
+    highly_relevant: List[str]
+    probably_relevant: List[str]
+    unranked_relevance: List[
+        str
+    ]  # When we don't know if the test is relevant to the PR or not
```

Comment:
any chance you could get the autoformatter to not make not look weird lol

Reply:
👹